src/Extensions/AttributeExtensions.cs

```diff
@@ -28,6 +29,11 @@ public static AttributeListSyntax WithStepAttribute(this SyntaxList<AttributeLis
             return list.First(syntax => GetStepAttribute(syntax.Attributes) != null);
         }
 
+        public static Boolean IsRecoverable(this MemberDeclarationSyntax method)
+        {
+            //TODO: Figure out if method is recoverable
+            return false;
```

Comment:
How are FTs passing if this is not implemented?

Reply:
FT are passing cause while execution, the AssemblyLoader is used to check if the step is recoverable.
For validation(static parsing) we don't need id step is recoverable. It can be removed.